src/javaharness/java/arcs/web/impl/WebHarnessController.java

```diff
@@ -64,6 +67,17 @@ public void init() {
                 environment.sendMessageToArcs(
                     "{\"message\": \"autofill\", \"modality\": \"dom\", \"entity\": {\"type\": \"address\"}}",
                     (id, result) -> dataParagraph.append("Test: " + result))));
+
+    document.body.appendChild(
```

Comment:
What does this do?

Reply:
it spawns an arc and instantiates "DemoText" recipe in it that renders "hello world" (in `log` and `alert` modalities) - i demo-ed it in standup last week.